inductiva/fluids/scenarios/coastal_area/coastal_area.py

```diff
@@ -17,6 +19,8 @@
                                        replace_params_in_template)
 from inductiva.fluids.scenarios.coastal_area.output import CoastalAreaOutput
 
+from inductiva.generative import diamond_square
```

Comment:
Maybe you want to take the chance to correct the import style as mentioned by Hugo. We should avoid directly importing function/classes.